code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -71,13 +96,15 @@
 	icon = 'icons/effects/fire.dmi'
 	icon_state = "1"
 	layer = GASFIRE_LAYER
-	plane = ABOVE_GAME_PLANE
```

Comment:
Wait, is this change to the plane intended?

Reply:
No, must've been left over from wallening